bambi/models.py

```diff
@@ -517,6 +518,17 @@ def set_alias(self, aliases):
                 for term in self.response_component.group_specific_terms.values():
                     if name in term.prior.args:
                         term.hyperprior_alias = {name: alias}
+
+                if (
```

Comment:
Add a comment to this one and the code block below so when reading this module its more clear how the list, and the `if missing_name` block below interact

Reply:
done

Reply:
Thanks! Tomas can comment on the rest of the logic. LGTM but he knows best!

Reply:
If I understand correctly this is "asking" the same questions that were in the for loop again. If they are all False, it means none of the if-blocks in the previous if-statements got executed, implying the alias was not used. 

What do you think about this alternative approach (pseudo-code)?


```python
for name, alias in aliases.items():
    is_used = False
    
    if cond1:
        ....
        is_used = True

    if cond2:
        ...
        is_used = True


     # finally
     if not is_used:
          missing_names.append(name)
```

I know it introduces a new auxiliary variable. But I think it's slightly clearer since the human-reader doesn't need to parse a "complex" if statement with multiple conditions.

Reply:
Definitely cleaner. Added.